linode/manager
# hotfix: Revert Linode Landing Changes

## Description 📝

- Manually reverts #9062

## How to test 🧪

> **Warning**: Please test Linodes Landing thoroughly 

- Test functionality of Linodes Landing
- Test action menu actions
  - Please verify that actions and their respective dialogs work
- Test Group by tag and summary view
- Verify that the table still updates in real time with percentages from events data
